### ISSUE TITLE
Changing default gnome-terminal theme

### ISSUE THREAD

**Issue by mjgritli** [regolith-linux/regolith-styles#36]
Dear Developer,

I love your work of integration between i3 and ubuntu-gnome. However, 
I couldn't customize my gnome-terminal, everytime I do, it gets overridden when I refresh my i3 config or re-login.

Why do I have to stick to Regolith as the default profile to use.
Where can I override this?
btw, I'm on Regolith unstable, 1.5.1

Regards

**Comment by naib864** [comment]
I had this problem too, atm the workaround is to go to /etc/regolith/styles and edit the st-term file. There every terminal color is defined to take the colors definied by the current style. You can replace the colors with the RGB values of your prefered profile.

**Comment by kgilmer** [comment]
There are two places that the gnome-terminal profile is changed:
1. upon first login of a regolith session
2. every time `regolith-look` is run, which includes when i3 is refreshed.

For #1 there is no way of disabling this first profile change in Regolith 1.5.1.  However for #2 (the one that matters), if you set `gnome.terminal.update` to false then the profile will not be updated.

So, something like this should resolve your issue: 

```bash
$ echo "gnome.terminal.update: false" >> ~/.config/regolith/Xresources
```

**Comment by kgilmer** [comment]
Forgot to reference this issue in the commit message, so here it is: https://github.com/regolith-linux/regolith-ftue/commit/3bf017752f695f3c874cb94780973846e29b8e9a

**Comment by kgilmer** [comment]
Resolved in 1.5.2
